registers/TUTORIAL.md

```diff
@@ -0,0 +1,16 @@
+Step 1
+------
+
+We need a source of data that we are going to log. This could be
+anything, but for this tutorial, I've chosen the [GDS Registers
+project](https://www.gov.uk/government/publications/registers/registers).
+
+In this step, we start with a simple program that reads a register and
+dumps its contents to stdout.
+
+You can run it with
+
+```go run dump/main.go```
```

Comment:
Does this work without the ./ i.e:`go run ./dump/main.go`?

Also you only need single backticks as it's all on one line.

Reply:
It does. No idea why you always include the . :-)

Reply:
Everything seems to break for me if I don't include the ./ e.g. go test :/